ocaml/xapi/xapi_vm_group_helpers.mli

```diff
@@ -0,0 +1,32 @@
+(*
+ * Copyright (C) 2024 Cloud Software Group
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+val update_vm_placement_alert_with_vm :
```

Comment:
As the alert is specific for "VM anti-affinity rules", I think it is better to name these functiions as `_vm_anti_affinity_alert_`.

Reply:
All related function names have been updated